tailwind.config.ts

```diff
@@ -7,7 +7,7 @@ const config = {
     "./pages/**/*.{ts,tsx}",
     "./components/**/*.{ts,tsx}",
     "./app/**/*.{ts,tsx}",
-    "./src/**/*.{ts,tsx}",
+    "./lib/number.tsx",
```

Comment:
Small note, for the classes being used in the number.tsx file, we ned to include that file or the tailwind compiler will miss them and not include these classes.